packages/expo-cli/src/commands/codesigning/configure.ts

```diff
@@ -0,0 +1,14 @@
+import type { Command } from 'commander';
+
+import { applyAsyncActionProjectDir } from '../utils/applyAsyncAction';
+
+export default function (program: Command) {
+  applyAsyncActionProjectDir(
+    program
+      .command('codesigning:configure [path]')
+      .description('Configure and validate expo-updates code signing for this project')
```

Comment:
Just to confirm, this says expo-updates (the library) and not Expo Updates (the spec) because it configures app.json for expo-updates?

Reply:
Correct, expo-updates prescribes the certificate format and fields in app.json for use by the config plugin to configure the native configuration.

The spec doesn't actually prescribe anything but the use of certificates and private keys to sign manifests. i.e. it doesn't specify how to get the certificate into the app or whether the expo-updates library needs to be used.